package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/refocus-sample-generator-template-utils",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
updating version manually or as part of npm publish?

Reply:
Did you see my other comment? npm publish does not automatically bump the version - it assumes that package.json already has the updated version. In any case, we want the version in GitHub to match the version in npm, so this needs to be checked in here.